### PR TITLE
Hotfix: 1.32.1 rosetta zero balance account

### PR DESCRIPTION
**Pull request by encody**

### CODE REVIEW

File: chain/rosetta-rpc/src/utils.rs [hunk 1]
```diff
@@ -274,9 +274,14 @@ fn get_liquid_balance_for_storage(
     account: &near_primitives::account::Account,
     storage_amount_per_byte: near_primitives::types::Balance,
 ) -> near_primitives::types::Balance {
-    let required_amount =
-        near_primitives::types::Balance::from(account.storage_usage()) * storage_amount_per_byte;
-    required_amount.saturating_sub(account.locked())
+    let staked_for_storage = if account.storage_usage() <= 770 {
```
Comment (jakmeier) [inline chain/rosetta-rpc/src/utils.rs:277]: nit: instead of using a magic 770, maybe make it named constant or maybe create a function `is_zero_balance_account`?

(Assuming a dependency on `node-runtime` is not an option, otherwise you could important `ZERO_BALANCE_ACCOUNT_STORAGE_LIMIT` or even make `is_zero_balance_account` pub and import that. This could prevent similar errors in the future.)

**Review by jakmeier — APPROVED** [review]
Looks good, although I'd prefer the hard coded number to be a bit less magic. :) Optionally change it as you see fit and then add the auto-merge label to get it merged.

Question for a potential follow up PR: Would it make sense to add a Rosetta RPC test that checks it knows how to deal with zero balance accounts?

**Comment by jakmeier** [comment]
Oh and I see you changed it to merge into the 1.32.1 branch instead of master.  @wacban is this the right order? Or should it be first merged into master, and then cherry-picked into release branches?

**Comment by encody** [comment]
Since I branched this off of 1.32.1 tag, I have to make a new PR so that the history doesn't contain the 1.32.1-specific changes.

**Comment by jakmeier** [comment]
> Since I branched this off of 1.32.1 tag, I have to make a new PR so that the history doesn't contain the 1.32.1-specific changes.

Either that or rebase/cherry-pick and force-push, it shouldn't matter which way you do it. Just add me as a reviewer if you create a new PR so I can approve that one as well.

**Review by jakmeier — CHANGES_REQUESTED** [review]
needs to go to a different branch

**Comment by encody** [comment]
Use https://github.com/near/nearcore/pull/8830 instead
